providers/php_agent.rb

```diff
@@ -25,11 +26,16 @@
 
   # ensure that the file #{new_resource.daemon_upgrade_file}
   # does not exist if it does, move it aside (or remove it)
-  execute 'newrelic-backup-key' do
-    command "mv #{new_resource.daemon_upgrade_file} #{new_resource.daemon_upgrade_file}.external"
+  file "#{new_resource.daemon_upgrade_file}.external" do
+    content ::File.open(new_resource.daemon_upgrade_file).read
```

Comment:
This will be evaluated on compile time, won't it? It will then fail when the file is not present upon chef start.